pkg/operator/configobserver/apiserver/observe_tlssecurityprofile_test.go

```diff
@@ -29,7 +28,17 @@ func TestObserveTLSSecurityProfile(t *testing.T) {
 			name:                  "NoAPIServerConfig",
 			config:                nil,
 			expectedMinTLSVersion: "VersionTLS12",
-			expectedSuites:        crypto.OpenSSLToIANACipherSuites(configv1.TLSProfiles[configv1.TLSProfileIntermediateType].Ciphers),
+			expectedSuites: []string{
+				"TLS_AES_128_GCM_SHA256",
+				"TLS_AES_256_GCM_SHA384",
+				"TLS_CHACHA20_POLY1305_SHA256",
+				"TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+				"TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
+				"TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
+				"TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
+				"TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256",
+				"TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256",
+			},
```

Comment:
Could you explain why we need to hardcode the suites now?

Reply:
The tests were using the code under test to generate the lists. So, for example, if I had added the wrong suites, the tests would not catch it.

Reply:
Makes sense, thanks